### ISSUE TITLE
bricked device

### ISSUE THREAD

**Issue by SebiTimeWaster** [OtherCrashOverride/odroid-go-firmware#11]
hi, i tried to flash the newest firmware on linux, but i didnt run "eraseflash", i just ran "flashall". 
flashing seemed fine, but now after booting i get "no files error", and i cannot flash again, trying to connect just results in a boot loop. 

1. what happened? is it because i didnt erase first? 
2. is there a way to recover?

**Comment by OtherCrashOverride** [comment]
Related: https://github.com/espressif/esp-idf/issues/2218

Erasing the flash before re-flashing will restore normal operation.

**Comment by SebiTimeWaster** [comment]
thx for your answer, unfortunately i cannot erase the flash since the script cannot connect to the device (because of the aforementioned boot loop).

**Comment by SebiTimeWaster** [comment]
also it may be a good idea to combine both scripts into one to avoid people running into the same problem...

**Comment by SebiTimeWaster** [comment]
i was now able to erase flash and reflash the firmware under os-x, but i still get the same "no files error" as before.

**Comment by OtherCrashOverride** [comment]
The most likely issue is that /odroid/data/col and /roms/col are not present on the SD card.

**Comment by SebiTimeWaster** [comment]
no, i added them beforehand. i just re-checked that they are there, still same error.

**Comment by OtherCrashOverride** [comment]
Please post a picture of the error so that it can be identified.  You can also try using a different SD card.

**Comment by SebiTimeWaster** [comment]
![odroid_error](https://user-images.githubusercontent.com/3726309/43641659-d1ba6c20-9724-11e8-9421-d2fb514a4701.jpg)

the sd-card is fine, it works perfectly in my other Odroid Go i didn't update yet...

**Comment by OtherCrashOverride** [comment]
That error indicates there are no files in the /odroid/firmware folder of the SD card.

The emulator application is located here:
https://github.com/OtherCrashOverride/go-play/releases/download/20180726-smsplusgx/Go-Play.fw

Download the above "Go-Play.fw" file and copy it to the /odroid/firmware folder of the SD card.

**Comment by SebiTimeWaster** [comment]
that worked.

so is this procedure (that is documented nowhere) replacing the old procedure of placing a "firmware.bin" file in the root directory and pressing B+turn on?

**Comment by kamotswind** [comment]
It is on YouTube :) Search for odriod go firmware update

This firmware is still considered prerelease I think so the wiki hasn't been updated yet.

**Comment by OtherCrashOverride** [comment]
From the 20180826 release on, applications will go in the /odroid/firmware folder of the SD card.  You can find information regarding this firmware on the Odroid forums:
https://forum.odroid.com/viewtopic.php?f=158&t=31513
https://forum.odroid.com/viewtopic.php?f=159&t=31348#p229849

A video is here:
https://www.youtube.com/watch?v=2LD-BfR9-MI

**Comment by OtherCrashOverride** [comment]
> that worked.

Closing.

**Comment by SebiTimeWaster** [comment]
How about writing a nice little MD file here that documents flashing / usage instead of giving the user information on 5 different forum entries, 3 wiki pages, a youtube video and 2 websites?

**Comment by OtherCrashOverride** [comment]
Currently the software is intended for early adopters and enthusiasts.  As noted earlier, things change dramatically during this stage of development.  Once finalized, documentation will be produced.

The rational is that as I am the only person working on this in my personal time, I have to optimize the use of my time and resource which prioritize development over documenting interim releases.

**Comment by OtherCrashOverride** [comment]
TL;DR = Lone developer, limited time.  Community help is greatly appreciated (like the video linked).
